lang/block.go

```diff
@@ -36,6 +38,13 @@ const (
 	ListBlock BlockKind = "list"
 )
 
+const urlPckgGoDev = "https://pkg.go.dev"
+
+var (
+	gfmWhitespaceRegex = regexp.MustCompile(`\s`)
+	gfmRemoveRegex     = regexp.MustCompile(`[^\pL-_\d]+`)
```

Comment:
Keeping in hindsight that regular expressions to often are a sort of *executable line noise* /s this regexp does not really explain itself to the reader.

Do you think it would help to add a comment of what strings are targeted by this regexp?